src/Actions/ManagesBlueprints.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Novu\SDK\Actions;
+
+use Novu\SDK\Resources\Blueprint;
+
+trait ManagesBlueprints
+{
+    /**
+     * Fetch list of blueprints [ Come back to this for pagination---->]
+     *
+     * @return Blueprint
```

Comment:
Use the full path in the @return. like this `@return Novu\SDK\Resources\Blueprint`